spec/ordersystem_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'ordersystem'
+
+describe OrderSystem do
+  let(:dish) { double :dish }
+  let(:option_one) { { dish: dish, price: 3 } }
+  let(:option_two) { { dish: dish, price: 5 } }
+  let(:takeaway) { TakeawayKitchen.new.menu }
+  
+  it 'returns the menu from TakeawayKitchen' do
+    expect(subject.menu).to eq takeaway
+  end
+
+  it 'initializes with an empty order' do
+    expect(subject.order).to be_empty
+  end
+
+  describe '#view_menu' do
+    it 'returns a hash' do
+      expect(subject.view_menu).to include({ dish: be_an(String), price: be_an(Integer) })
```

Comment:
The purpose of `view_menu` is to print, not to return things. Printing is what you should test for (using the output matcher from rspec)